OpenRA.Mods.Common/Widgets/RadarWidget.cs

```diff
@@ -85,7 +85,8 @@ public override void Initialize(WidgetArgs args)
 
 			// Set initial terrain data
 			foreach (var cell in world.Map.AllCells)
-				UpdateTerrainCell(cell);
+				if (world.Map.Contains(cell))
```

Comment:
Map.AllCells by definition only contains cells in the map.  This check is unnecessary and just shows down the game.